visual-inspection/inspection-frontend/src/App.vue

```diff
@@ -105,6 +104,12 @@ export default {
   },
   methods: {
     async toggleConfiguration() {
+      if (this.showConfiguration && !this.currentPrediction){
+        let count = 0
+        while(count < 2000000){
+          count += .01
```

Comment:
```suggestion
        await new Promise(resolve => setTimeOut(resolve, 500));
```

Das funktioniert so nicht richtig und ist keine gute Lösung. Besser einen Timer setzen.

Details siehe: https://stackoverflow.com/questions/951021/what-is-the-javascript-version-of-sleep

Reply:
Alternativ kannst du es auch über eine CSS-Transition lösen, so wie wir das im SpinningIndicator machen, den wir im Review-Sentiment-Use-Case einsetzen: https://github.com/XAI-Demonstrator/xai-demonstrator/blob/master/common/vue-components/xaidemo-ui/src/components/SpinningIndicator.vue

Reply:
Die setTimeOut Variante liefert mir im Browser folgenden Fehler:

---

./src/App.vue
Module Error (from ./node_modules/eslint-loader/index.js):

C:\...\xai-demonstrator\visual-inspection\inspection-frontend\src\App.vue
  108:38  error  'setTimeOut' is not defined  no-undef

✖ 1 problem (1 error, 0 warnings)

---

Reply:
Das O muss klein :-)

Reply:
als ob ^^  
Danke dir!